pravega/pravega
# [issue-2155] Publish pravega snapshots to jcenter repo

**Change log description**
This PR addresses the issue https://github.com/pravega/pravega/issues/2155.. A new publishing configuration is added to address the publishing task to JCenter repository. Travis configurations are modified to trigger the snapshot publish *only* when a commit is pushed to the `master` branch

**Purpose of the change**
To make the Pravega master snapshots available for any dependent projects like `flink-connector` etc.,

**What the code does**
Allows Pravega artifacts to be published to snapshots repository.
https://oss.jfrog.org/artifactory/jfrog-dependencies/io/pravega/

**How to verify it**
./gradlew check publish -PpublishUrl=jcenterSnapshot -PpublishUsername=$BINTRAY_USER -PpublishPassword=$BINTRAY_KEY
